cypress/integration/formContextWithValidation.spec.js

```diff
@@ -0,0 +1,24 @@
+beforeEach(() => {
```

Comment:
It seems a case in which is better to use beforeAll over beforeEach

Reply:
beforeall don't exist on cypress